src/content/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github.mdx

```diff
@@ -0,0 +1,2103 @@
+        See [our documentation](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/) to learn about environments.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        eventType
+      </td>
+
+      <td>
+        String!
+      </td>
+
+      <td>
+        An issue event that could be INCIDENT_ADDED, USER_ACTION, INCIDENT_CLOSED, etc...
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        incidentIds
+      </td>
+
+      <td>
+        [String!]
+      </td>
+
+      <td>
+        A list of incident IDs in this issue. In some cases there may be thousands of incidents in an issue.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        isCorrelated
+      </td>
+
+      <td>
+        Boolean!
+      </td>
+
+      <td>
+        True means that other issues were correlated to this one.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        isIdle
+      </td>
+
+      <td>
+        Boolean!
+      </td>
+
+      <td>
+        See our documentation to learn about idle issues.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        issueId
+      </td>
+
+      <td>
+        ID!
+      </td>
+
+      <td>
+        Issue identifier.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        origins
+      </td>
+
+      <td>
+        [String!]!
+      </td>
+
+      <td>
+        A list of origins of incidents like violations, anomalies, external etc...
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        priority
+      </td>
+
+      <td>
+        Priority!
+      </td>
+
+      <td>
+        The issue priority. Can be one of: LOW, MEDIUM, HIGH, CRITICAL.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        sources
+      </td>
+
+      <td>
+        [String!]!
+      </td>
+
+      <td>
+        A list of sources of the incidents like newrelic or external.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        state
+      </td>
+
+      <td>
+        IssueState!
+      </td>
+
+      <td>
+        Issue state can be one of: `CREATED`, `ACTIVATED`, `DEACTIVATED`, `CLOSED`.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        title
+      </td>
+
+      <td>
+        [String!]!
+      </td>
+
+      <td>
+        A list of underlying incident titles.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        totalIncidents
+      </td>
+
+      <td>
+        Int!
+      </td>
+
+      <td>
+        The count of underlying incidents.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        updatedAt
+      </td>
+
+      <td>
+        EpochMilliseconds!
+      </td>
+
+      <td>
+        The time the issue was last updated.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+### Issue lifecycle and state [#issue-lifecycle]
+
+Idle issues:
+* An issue can become idle if it is too big, meaning it contains 5000 incidents or more.
+* An idle issue does not accept any more incidents.
+* An idle issue will not correlate to other issues.
+
+## Incident type [#incident-type]
```

Comment:
as a general note, for any major header, it's good to have some explanatory text. for example, in this case, giving some context about what the following table is meant to show and why it's important.